docs/metadao/faq.md

```diff
@@ -0,0 +1,35 @@
+# Frequently Asked Questions (FAQ)
+
+## 1. Futarchy Questions
+
+### Q: Will futarchy always be correct?
+**A:** Sometimes futarchy may result in decisions that seem suboptimal in hindsight. However, it remains one of the most effective governance mechanisms available.
```

Comment:
I would call out that no decision-making system can always be correct as long as we live in a probabilistic world

Reply:
Beautiful. Changed in PR